### PR TITLE
Fix typo: Perfect Forward Security is PFS instead of FPS

### PR DESCRIPTION
**Pull request by TimWolla**

### CODE REVIEW

File: proto/proto_handshake.c [hunk 1]
```diff
@@ -61,11 +61,11 @@ handshakefail(struct handshake_cookie * H)
 }
 
 /**
- * proto_handshake(s, decr, nofps, requirefps, K, callback, cookie):
+ * proto_handshake(s, decr, nopfs, requirepfs, K, callback, cookie):
  * Perform a protocol handshake on socket ${s}.  If ${decr} is non-zero we are
  * at the receiving end of the connection; otherwise at the sending end.  If
- * ${nofps} is non-zero, perform a "weak" handshake without forward perfect
- * secrecy.  If ${requirefps} is non-zero, drop the connection if the other
+ * ${nopfs} is non-zero, perform a "weak" handshake without forward perfect
```
Comment (gperciva) [inline proto/proto_handshake.c]: missing fix: "forward perfect \n secrecy"

File: proto/proto_handshake.h [hunk 2]
```diff
@@ -6,11 +6,11 @@ struct proto_keys;
 struct proto_secret;
 
 /**
- * proto_handshake(s, decr, nofps, requirefps, K, callback, cookie):
+ * proto_handshake(s, decr, nopfs, requirepfs, K, callback, cookie):
  * Perform a protocol handshake on socket ${s}.  If ${decr} is non-zero we are
  * at the receiving end of the connection; otherwise at the sending end.  If
- * ${nofps} is non-zero, perform a "weak" handshake without forward perfect
- * secrecy.  If ${requirefps} is non-zero, drop the connection if the other
+ * ${nopfs} is non-zero, perform a "weak" handshake without forward perfect
```
Comment (gperciva) [inline proto/proto_handshake.h]: missing fix: "forward perfect \n secrecy"

**Comment by cperciva** [comment]
My first thought was that this was a rather sweeping change for a cosmetic fix, but I suppose "perfect forward secrecy" is the more common spelling... yeah, consider this approved in principle but I'd like the places @gperciva noticed fixed at the same time.

**Comment by TimWolla** [comment]
> My first thought was that this was a rather sweeping change for a cosmetic fix, but I suppose "perfect forward secrecy" is the more common spelling...

I thought it was a simple typo in the variable name, because I stumbled upon it when reading the code. I did not expect that you would call it “forward perfect” in the long form, as I never heard it that way before.

Anyway: I fixed the places @gperciva mentioned and force pushed my branch.

**Comment by gperciva** [comment]
Thanks, LGTM!

**Comment by TimWolla** [comment]
> Thanks, LGTM!

You should press the “approve changes” link in the box with the merge button then, if you intend to use the review feature in the future.

**Comment by gperciva** [comment]
@TimWolla Sorry, thanks for the tip!

**Pull request merged**